pkg/logging/logging.go

```diff
@@ -71,8 +71,13 @@ var (
 		"info":    logrus.InfoLevel,
 		"debug":   logrus.DebugLevel,
 	}
+
+	options = LogOptions{}
```

Comment:
This feels a little clunky. Maybe it's just that it is called options instead of logOpts (to contrast with sysLogOpts). I was a little confused why `ConfigureLogLevel` didn't take it as a variable since it took a debug toggle, but that it used `options` and it seemed liked a local variable.

Reply:
Do you want me to call it `logOptions`? I'm confused as to what you would like me to change here.

Reply:
Yeah, I think that's the only actionable part of my rant :/ Lets rename it despite the stutter with the type